src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -657,9 +657,8 @@ private SiloHandle StartOrleansSilo(Silo.SiloType type, TestingSiloOptions optio
         /// <param name="type">The type of the silo to deploy</param>
         /// <param name="options">The options to use for the silo</param>
         /// <param name="instanceCount">The instance count of the silo</param>
-        /// <param name="shared">The shared AppDomain to use</param>
         /// <returns>A handle to the deployed silo</returns>
-        public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType type, TestingSiloOptions options, int instanceCount, AppDomain shared = null)
+        public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType type, TestingSiloOptions options, int instanceCount)
```

Comment:
@jdom about the last comment. The TestingSiloHost is used by some tests in TesterAzureUtils. I didn't remember we talked about not porting it over, but I know it is being deprecated. 
So two option we got here: 
1. don't port it over, remove those tests using it in TesterAzureUtils too and remember which tests are those.
2. port it over, migrate it off in both main and vNext. 
I prefer option 2 so that we don't keep a list somewhere about which tests still need to port over to vNext. 
But if you are strong about option 1, I can do it. 
About your concern of this change would break tests in VSO. I tested with NightlyFunc. It didn't break